src/app/api/graphql/route.js

```diff
@@ -0,0 +1,65 @@
+import { startServerAndCreateNextHandler } from '@as-integrations/next'
+import { ApolloServer } from '@apollo/server'
+import {
+  ApolloServerPluginLandingPageLocalDefault,
+  ApolloServerPluginLandingPageProductionDefault,
+} from '@apollo/server/plugin/landingPage/default'
+import { gql } from 'graphql-tag'
+import { MongoClient } from 'mongodb'
+
+// The connection string for mongodb connection.
+const uri =
+  'mongodb+srv://ayushtiwari110:Ud7JOzOQRw6J93W2@bacpac.i8d4g8c.mongodb.net/'
```

Comment:
We'll prefer to store the string as the environment variable in GitHub.
Locally you can test it by creating a `.env` file and store this URL over there.

Here's the link for the GitHub docs - https://docs.github.com/en/actions/learn-github-actions/variables

Reply:
Yeah, I thought once I create bacpac MongoDB account, then i'll add that directly in .env file.